uiscene_patch.py

```
@@ -0,0 +1,73 @@
+import sys
+
+def patch_uiscene():
+    path = 'src/scripts/scenes/UIScene.js'
+    with open(path, 'r') as f: content = f.read()
```

Ruff (0.15.2)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)